### PR TITLE
Create american-journal-of-enology-and-viticulture.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/69219/style-request-american-journal-of-enology-viticulture

The template style was made by me. Must be old. Jesus... I need to fix the old, too. So many weird things in there. I've gotten the hang of this now... Just not Chicago styles.... haha.

### CODE REVIEW

File: american-journal-of-enology-and-viticulture.csl [hunk 1]
```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
```
Comment (rmzelle) [inline american-journal-of-enology-and-viticulture.csl]: @POBrien333, can you check the locale? The guide says American-English, but I don't know if "en-US" would e.g. mess up the quotes.

File: american-journal-of-enology-and-viticulture.csl [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    <id>http://www.zotero.org/styles/american-journal-of-enology-and-viticulture</id>
+    <link href="http://www.zotero.org/styles/american-journal-of-enology-and-viticulture" rel="self"/>
+    <link href="http://www.zotero.org/styles/friedrich-schiller-university-jena-faculty-of-medicine" rel="template"/>
+    <link href="http://www.ajevonline.org/content/authors" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="medicine"/>
```
Comment (rmzelle) [inline american-journal-of-enology-and-viticulture.csl]: @POBrien333, wine is not really a medicine ;).

Comment (adam3smith) [reply american-journal-of-enology-and-viticulture.csl]: well... https://doi.org/10.4103%2F0975-3583.74259

Comment (rmzelle) [reply american-journal-of-enology-and-viticulture.csl]: "All Breathlessness In Mitral Stenosis Is Not Pulmonary Edema- A Case of Mitral Stenosis with Cardio Respiratory Arrest" ? Don't see the wine connection.

Comment (adam3smith) [reply american-journal-of-enology-and-viticulture.csl]: oops copied the DOI from PMC. Was supposed to be this one: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3023893/

Comment (rmzelle) [reply american-journal-of-enology-and-viticulture.csl]: Better, but my complaint stands :).

(I reported the DOI problem at https://support.crossref.org/hc/en-us/requests/new. It should resolve to http://old.jcdronline.org/article/145 but resolves to http://www.jcdronline.org/article/145. Note the identical article no.)

File: american-journal-of-enology-and-viticulture.csl (hunk 1 above)
Comment (POBrien333) [reply american-journal-of-enology-and-viticulture.csl]: Thanks. Checked and indeed, en-US is better a fit.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**